### DISCUSSION TITLE
As Admin I cannot reset the MFA for a User in case of missing possiblility to login

### DISCUSSION THREAD

**Discussion by BotKup** [umbraco/Umbraco-CMS#15612] [Features and ideas]
Hi

I have added a backend user to the Umbraco backend. 
The user has set up his MFA authentication on the profile editing.
The user has lost the phone and cannot log in anymore to the Umbraco backend.
User has requested to reset the MFA to can use the Umbraco backend.

The Umbraco support said that I need to add the user as Member with Readonly rights and then I can do the reset.
I invited the user as Member, he got the invitation email.
The only problem that the user needs to accept the invitation as member, but for accepting the user needs to log in to the backend. So this is not possible.
As Admin I can reset MFA only for the members which has accepted the invitation.

So finally this is a circular reference and it is not possible to do it without the Umbraco Support.

As Admin I should be able to reset the MFA for users in the umbraco User management easily!

Regards
Botond Kupan

**Comment by bergmania** [comment]
Hi @BotKup ..

I'm sorry, but Im not sure I follow..

As admin you should be able to just remove the 2FA from the other user. Thereby the user can sign-in and activate 2FA again.

**Reply by BotKup** [reply]
For the backens user there is no possibility for admin to do that. The user can do that on his profile page. But he cannot log in.

**Comment by nul800sebastiaan** [comment]
Is this an Umbraco Cloud request perhaps?

**Reply by BotKup** [reply]
Yes. It is Cloud

**Comment by sajumb** [comment]
Hi @BotKup,
You're absolutely correct in your assessment. The current process does have its limitations, particularly in situations where a user loses access to their MFA device. Our support team's suggestion to add the user as a member with ReadOnly rights was intended as a workaround, but as you've pointed out, it isn't viable if the user is unable to log in to accept the invitation.

I apologize for the inconvenience this has caused. We acknowledge that the process is not ideal, and I want to reassure you that we have already recognized this as an area for improvement. In fact, enhancing the MFA reset flow is on our backlog, and we are actively working towards making this process more streamlined and user-friendly.

In the meantime, to assist with this specific case, please send an email to my Umbraco email starting with 'saj'. I will prioritize the reset of the user's MFA settings to ensure they regain access as quickly as possible, provided it hasn't already been reset.

kind regards
Søren, PO at Umbraco Cloud

**Comment by NurhakKaya** [comment]
I am experiencing this problem and it seems like contacting Umbraco is still the only option to reset the MFA settings of the Umbraco Cloud user, and this is not really a feasible solution for a paid product, such as Umbraco Cloud. 

Is there a plan to enhance the MFA process by adding additional verification options, such as SMS or email?

PS: saj@umbraco.com and saj@umbraco.dk email addresses don't seem to be working.

<img width="1118" height="668" alt="image" src="https://github.com/user-attachments/assets/b949b445-3ee0-4baa-9766-d20ea8949b30" />

**Comment by AndyButland** [comment]
You should contact support for this @NurhakKaya to get an update, or raise it on the [Cloud issue tracker](https://github.com/umbraco/Umbraco.Cloud.Issues).  I'll close this discussion on the CMS repository to avoid future confusion.
